server/routes/TaskRoutes.js

```diff
@@ -5,11 +5,14 @@ const router = express.Router();
 
 // Create task route
 router.post('/create', async (req, res) => {
+  console.log('Creating task for user:');
   const { username, taskType, date, startTime, endTime, duration, points } = req.body;
+  console.log('Creating task for user with these details:', req.body);
   const user = await UserModel.findOne({ username });
   if (!user) {
-    return res.json({ message: 'Invalid user' });
+    return res.status(401).json({ message: 'Invalid user' });
   }
+  console.log('Creating task for user:', user);
```

Comment:
Can you either remove this or change it to be user.username or something to make the output less verbose